tools/deb/istio-start.sh

```diff
@@ -44,8 +40,20 @@ if [ -z "${POD_NAME:-}" ]; then
   POD_NAME=$(hostname -s)
 fi
 
+
+# Init option will only initialize iptables. Can be used
+if [[ ${1-} == "init" || ${1-} == "-p" ]] ; then
+  # Update iptables, based on current config. This is for backward compatibility with the init image mode.
+  # The sidecar image can replace the k8s init image, to avoid downloading 2 different images.
+  ${ISTIO_BIN_BASE}/istio-iptables.sh $*
```

Comment:
should be "${@}"
but I'm concerned we're turning this into a multi command error prone shell script

Reply:
please fix $* it will bug out with spaces in params

Reply:
Done, thanks for the info - didn't know about this.